src/error.rs

```diff
@@ -74,6 +74,9 @@ pub enum Error {
     #[cfg(feature = "functions")]
     #[allow(dead_code)]
     UserFunctionError(Box<error::Error + Send + Sync>),
+
+    /// Error available for the implementors of the `FromSql` trait.
+    Other(Box<error::Error + Send + Sync>),
```

Comment:
The PR said this was for `ToSql` but this comment says it's for `FromSql`. There is already one error case for `FromSql` conversion errors. Could you confirm if this was for `ToSql` errors or `FromSql` errors?

I'd prefer a name with more clarity than `Other`. If this is for `ToSql`, maybe something like `ToSqlConversionFailure`?